jobs/dcp1-version/docker-app/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM quay.io/ebi-ait/ingest-base-images:python_3.7-slim
+
+COPY scripts/populate_dcp_version.py populate_dcp_version.py
+COPY scripts/requirements.txt requirements.txt
```

Comment:
move this above the py file to optimise build time

Reply:
Thanks!